packages/forms/resources/views/components/field-wrapper/index.blade.php

```diff
@@ -39,7 +39,16 @@
 
                 @if ($hint || $hintIcon)
                     <x-forms::field-wrapper.hint :icon="$hintIcon">
-                        {{ filled($hint) ? ($hint instanceof \Illuminate\Support\HtmlString ? $hint : \Illuminate\Support\Str::of($hint)->markdown()->sanitizeHtml()->toHtmlString()) : null }}
+                        @switch($hint)
+                            @case($hint instanceof \Filament\Forms\Components\Actions\Action)
+                            {{ $hint }}
+                            @break
+                            @case($hint instanceof \Illuminate\Support\HtmlString)
+                            {{ $hint }}
+                            @break
+                            @default
+                            {{ \Illuminate\Support\Str::of($hint)->markdown()->sanitizeHtml()->toHtmlString() }}
+                        @endswitch
```

Comment:
Wouldn't it be cleaner to use `@if` and `@elseif` here instead?

Reply:
I guess that is a matter of taste. Do you want me to change it? Will add a null case though, to break early.

Reply:
Personally I don't like nested if, elsif, else

Reply:
and if not null before

Reply:
This is no problem for me at all. I'll change it. It's just a matter of taste, what you and I find most readable.

Reply:
Yeah, here I don't see why you'd use a `switch` statement if you're writing an entire condition in the `case`s anyway. Also, if-else-if-else is more readable here.